nuxeo-drive-core/src/main/java/org/nuxeo/drive/service/SynchronizationRoots.java

```diff
@@ -61,4 +61,17 @@ public String getRepositoryName() {
         return refs;
     }
 
+    /**
+     * @since 9.1
+     */
+    @Override
+    public String toString() {
+        return new StringBuilder("repo = ").append(repositoryName)
+                                           .append(", paths = ")
+                                           .append(paths)
+                                           .append(", refs = ")
+                                           .append(refs)
+                                           .toString();
```

Comment:
Traditionally a debug `toString` method would include the class name and the list of fields after that inside parentheses or brackets: `SynchronizationRoots(repo = default, paths = ..., refs = ...)`